addon/modifiers/clipboard.js

```diff
@@ -27,6 +28,10 @@ export default modifier(function clipboard(element, params, hash) {
     element.setAttribute('data-clipboard-target', target);
   }
 
+  if (isBlank(element.id)) {
+    element.setAttribute('id', guidFor(element));
```

Comment:
Maybe this should be a `data-clipboard-id`? Definitely noteworthy that it will set and id if one is not provided